app/models/katello/glue/pulp/repo.rb

```diff
@@ -703,9 +703,9 @@ def full_path(smart_proxy = nil, force_https = false)
       end
     end
 
-    def index_content
+    def index_content(full_index = false)
       if self.yum?
-        Katello::Rpm.import_for_repository(self)
+        Katello::Rpm.import_for_repository(self, full_index)
```

Comment:
note if you do combine those two arguments as i mentioned above, this is then fine.